fsharp-backend/tests/Tests/EventQueueV2.Tests.fs

```diff
@@ -0,0 +1,172 @@
+module Tests.EventQueueV2
+
+open System.Threading.Tasks
+open FSharp.Control.Tasks
+
+open NodaTime
+
+open Expecto
+
+open Prelude
+open Prelude.Tablecloth
+open Tablecloth
+
+open Npgsql.FSharp
+open Npgsql
+open LibBackend.Db
+
+open TestUtils.TestUtils
+
+module PT = LibExecution.ProgramTypes
+module RT = LibExecution.RuntimeTypes
+module EQ = LibBackend.EventQueueV2
+module Canvas = LibBackend.Canvas
+module Serialize = LibBackend.Serialize
+
+module TI = LibBackend.TraceInputs
+module TFR = LibBackend.TraceFunctionResults
+
+
+let p (code : string) = FSharpToExpr.parsePTExpr code
+
+// This doesn't actually test input, since it's a cron handler and not an actual event handler
+
+let initializeCanvas (name : string) : Task<Canvas.Meta * tlid> =
+  task {
+    // set up handler
+    let! meta = initializeTestCanvas (Randomized name)
+
+    let h = testCron "test" PT.Handler.EveryDay (p "let data = Date.now_v0 in 123")
+    let oplists = [ handlerOp h ]
+
+    do!
+      Canvas.saveTLIDs
+        meta
+        [ (h.tlid, oplists, PT.Toplevel.TLHandler h, Canvas.NotDeleted) ]
+
+    return meta, h.tlid
+  }
+
+let enqueue (meta : Canvas.Meta) : Task<unit> =
+  let input = RT.DNull // I don't believe crons take inputs?
```

Comment:
```suggestion
  let input = RT.DNull // FSTODO I don't believe crons take inputs?
```

Reply:
Yeah, they don't take inputs so I'll just update the comment